docs/migration.md

```diff
@@ -0,0 +1,128 @@
+# Migrate from PostgreSQL Community to Percona Distribution for PostgreSQL 
+
+
+Percona Distribution for PostgreSQL provides the PostgreSQL database and the tools, add-ons, and extensions in a package that is guaranteed to work together and is easy to deploy.
+
+We encourage users to migrate from their PostgreSQL deployments to Percona Distribution for PostgreSQL. This document provides the migration instructions. 
+
+Depending on your business requirements, you may migrate to Percona Distribution for PostgreSQL either [on the same server](#migrate-on-the-same-server) or [onto a different one](#migrate-on-a-different-server). 
+
+## Migrate on the same server
+
+=== "On Debian and Ubuntu Linux"
+
+     1. Back up your data. The backup process is out of scope of this document. You can use `pg_dumpall` or other tools of your choice. 
+     2. Stop the `postgresql` server   
+
+         ```sh
+         $ sudo systemctl stop postgresql.service
+         ```
+
+     3. Remove community packages
+
+         ```sh
+         $ sudo apt-get --purge remove postgresql
+         ```
+
+      4. [Install percona-release](https://docs.percona.com/percona-software-repositories/installing.html)
+      5. Enable the repository
+
+           ```sh
+           $ sudo percona-release setup ppg14
+           ```
+
+      6. [Install Percona Distribution for PostgreSQL packages](installing.md#install-percona-distribution-for-postgresql-packages)
+      7. Restore the data from the backup.
```

Comment:
OK; so the backup/restore steps seem to be essential parts of the migration and there's no "in place" replacement possible? In that case, I think we should be more elaborate/explicit about how the backup and restore should be performed. This is definitely not out of the scope of this document then ;)